packages/fxa-shared/subscriptions/configuration/plan.ts

```diff
@@ -57,7 +57,7 @@ export class PlanConfig implements BaseConfig {
 
   // Extended by PlanConfig
   stripePriceId?: string;
-  productSet?: string[];
+  productSet!: string[];
```

Comment:
Although there isn't a linter rule for no-non-null-assertion in fxa-shared yet, we shouldn't be using it

Reply:
Ah, I see. Ignore this comment since it's not something easily fixable on your part here. We're misusing constructing this class below (see fromFirestoreObject).